provisioning/device/src/DeviceProvisioningClientException.cs

```diff
@@ -127,6 +82,11 @@ protected internal ProvisioningTransportException(SerializationInfo info, Stream
         /// </summary>
         public ProvisioningErrorDetails ErrorDetails { get; private set; }
```

Comment:
Odd that on provisioning we have this other class, whereas on hub we have the properties directly on the exception. We should look into making them look the same.